Backtracking/TugOfWar.js

```diff
@@ -0,0 +1,50 @@
+// https://www.geeksforgeeks.org/tug-of-war/
+
+// Given an array of n integers, we need to divide them into two subsets such that 
+// the difference between their sums is minimized.
+
+/**
+ * @param {number[]} arr - The array of integers.
+ * @returns {Array.<number[]>} Two subsets with minimized sum difference.
+ */
+
+
+function tugOfWar(arr) {
+    const n = arr.length;
```

Comment:
`n` is only used twice, and one usage can be eliminated => this variable should be removed, it doesn't help readability.